workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -111,6 +115,16 @@ public RedirectedProcess AddErrorProcessing(Action<string> errorProcessor)
             return this;
         }
 
+        /// <summary>
+        ///     Adds accumulated output processing
+        /// </summary>
+        /// <param name="errorProcessor">Processing action for error output.</param>
```

Comment:
this definition seems wrong

Reply:
Yup, copypasta. Thanks for the catch, I'll update this